drivers/scheduler/k8s/k8s.go

```diff
@@ -571,6 +592,24 @@ func (k *k8s) destroyCoreObject(spec interface{}, opts map[string]bool, app *spe
 		}
 
 		logrus.Infof("[%v] Destroyed StorkRule: %v", app.Key, obj.Name)
+	} else if obj, ok := spec.(*v1.Pod); ok {
+		if value, ok := opts[scheduler.OptionsWaitForResourceLeakCleanup]; ok && value {
+			pod, err := k8sOps.GetPodByName(obj.Name, obj.Namespace)
+			if err != nil {
+				logrus.Warnf("[%v] Error getting pods. Err: %v", app.Key, err)
+			}
+			podList = append(podList, pod)
```

Comment:
why not directly append to pods here?

Reply:
We get the following error in that case:
pods = append(pods, pod)
"first argument to append must be slice; have interface {}"